validation/models.py

```diff
@@ -403,6 +404,48 @@ def get_path_to_audio_directory() -> Path:
         return Path(settings.MEDIA_ROOT) / settings.RECVAL_AUDIO_PREFIX
 
 
+class Issue(models.Model):
+    comment = models.CharField(
+        help_text="The comment left by the validator",
+        blank=False,
+        max_length=1024,
+    )
+
+    other_reason = models.CharField(
+        help_text="A reason other than the options provided for why the issue was filed",
+        blank=True,
+        max_length=1024,
+    )
+
+    other = models.BooleanField(
+        help_text="There's another reason this issue was filed", default=False
+    )
+
+    bad_cree = models.BooleanField(
+        help_text="There's a better Cree word for this translation", default=False
+    )
+
+    bad_english = models.BooleanField(
+        help_text="There's another English word for this transcription", default=False
+    )
+
+    bad_recording = models.BooleanField(
+        help_text="There's one or more bad recordings in this batch", default=False
+    )
+
+    phrase = models.ForeignKey(Phrase, on_delete=models.CASCADE)
+
+    created_by = models.CharField(
+        help_text="The person who filed this issue",
+        default="",
+        max_length=64,
+    )
+
+    created_on = models.DateField(
+        help_text="When was this issue filed?", default=timezone.now()
```

Comment:
This line sets `created_on` to the date ~~_when the server was started_~~ **when the migration was run**. Luckily, Django provides `auto_now_add`, which I believe is the intention of the line:

```suggestion
        help_text="When was this issue filed?",
        auto_now_add=True,
```

See: https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.DateField.auto_now_add

Reply:
Ah, I think I added this because I already had some Issues locally, but didn't have the date on them. I can change it to auto_now_add, or I think you mention later in the comments that I can just take it out. Either way, this doesn't need to be in production--good catch!